sdk/react/discoveryPanel.js

```diff
@@ -93,6 +93,11 @@ var DiscoveryPanel = React.createClass({
   },
 
   render: function() {
+    var renderHorizontal = false;
+    if (Utils.shouldShowLandscape(this.props.width, this.props.height)) {
+      renderHorizontal = true;
```

Comment:
can we just use var renderHorizontal = Utils.shouldShowLandscape and returns true/false from shouldShowLandscape?